src/AppInstallerCommonCore/PackageVersionDataManifest.cpp

```diff
@@ -70,6 +72,14 @@ namespace AppInstaller::Manifest
         return "versionData.mszyml"sv;
     }
 
+    std::filesystem::path PackageVersionDataManifest::GetRelativeDirectoryPath(std::string_view packageIdentifier, std::string_view manifestHash)
+    {
+        std::filesystem::path result = "packages";

```

Comment:
static constexpr std::string_view?

Reply:
If it was to be used elsewhere, but this function is meant to prevent that need.